lib/zendesk_call.rb

```diff
@@ -8,7 +8,7 @@ def call(email)
 class GetTicketRequest
   def call(email, user_id)
     @user_id = user_id
-    @tickets = client.search(:query => "#{email}")
+    @tickets = client.search(:query => "#{email}", :reload => 'true')
```

Comment:
Actually, no it doesn't. I was wrong.
https://github.com/zendesk/zendesk_api_client_rb/wiki/Caching